### PR TITLE
Feat: KnowledgePage UI;Add React Icon; Link on HomePage; SideNav comp…

### PR DESCRIPTION
**Pull request by smarthulala**

### CODE REVIEW

File: src/components/SideNav/SideNav.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          <BiFile className="inline-block  text-blue-600 mr-2"></BiFile>
+          <span className="text-2xl">Report Builder</span>
+        </div>
+        <div className="mb-8 leading-10 ">
+          <ul>
+            <li>My Reports</li>
+            <li>Analytics</li>
+            <li>Knowledge Base</li>
+            <li>Help & Support</li>
+          </ul>
```
Comment (jeremyhuang035) [inline src/components/SideNav/SideNav.tsx:17]: can move to a new component called MenuItem or Item under SideNav/components

File: src/components/SideNav/SideNav.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { BiFile } from 'react-icons/bi'
+
+export default function SideNav() {
+  return (
+    <div className="bg-[#03111b] w-[305px] text-[#FFFFFF]">
+      <div className="mt-[55px] ml-[56px]">
+        <div className=" gap-x-3 mb-8 text-3xl">
```
Comment (jeremyhuang035) [inline src/components/SideNav/SideNav.tsx:7]:
```suggestion
        <div className="mb-8 text-3xl">
```

File: src/components/SideNav/SideNav.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { BiFile } from 'react-icons/bi'
+
+export default function SideNav() {
+  return (
+    <div className="bg-[#03111b] w-[305px] text-[#FFFFFF]">
```
Comment (jeremyhuang035) [inline src/components/SideNav/SideNav.tsx:5]: background color already defined in tailwind config file, e.g. bg-background, and try to use tailwind preset properties first, like text-white instead

**Review by jeremyhuang035 — COMMENTED** [review]
please check and remove all extra empty spaces in classnames

File: src/components/SideNav/SideNav.tsx (hunk 1 above)
Comment (smarthulala) [reply src/components/SideNav/SideNav.tsx]: Cool~

File: src/components/SideNav/SideNav.tsx (hunk 3 above)
Comment (smarthulala) [reply src/components/SideNav/SideNav.tsx]: Cool~

**Review by Qiming-Liu — APPROVED** [review]

**Review by Qiming-Liu — APPROVED** [review]

**Pull request merged**